package.json

```diff
@@ -48,6 +48,9 @@
     "unix-crypt-td-js": "^1.0.0"
   },
   "devDependencies": {
+    "@commitlint/cli": "^6.0.2",
+    "@commitlint/config-conventional": "^6.0.2",
+    "@commitlint/travis-cli": "^6.0.2",
```

Comment:
What it does `travis-cli`?

Reply:
this provides [commitlint-travis](https://github.com/marionebl/commitlint/tree/master/%40commitlint/travis-cli), travis-ci does not a full git clone so special handling is required.